test/parallel/test-buffer-fill.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
+const errors = require('../../lib/internal/errors');// Flags: --expose-internals
```

Comment:
Indeed. The `// Flags: --expose-internals` should be the first line of the test file,
The import must also be `require('internal/errors')`, without the `../../lib/` part.